Dockerfile

```
@@ -13,7 +13,7 @@ RUN xcaddy build \
     --with github.com/dunglas/vulcain/caddy
 
 # Prod image
-FROM php:8.3-fpm-alpine3.20 AS app_php
+FROM 8.4-fpm-alpine3.21 AS app_php
```

Hadolint (2.12.0)

[warning] Always tag the version of an image explicitly

(DL3006)